dataactbroker/handlers/fileHandler.py

```diff
@@ -702,15 +745,16 @@ def get_lines_from_csv(self, file_path):
     def load_d_file(self, url, upload_name, timestamped_name, job_id, isLocal):
         """ Pull D file from specified URL and write to S3 """
         job_manager = self.interfaces.jobDb
+        sess = GlobalDB.db().session
         try:
             full_file_path = "".join([CONFIG_BROKER['d_file_storage_path'], timestamped_name])
 
             _smx_logger.debug('Downloading file...')
             if not self.download_file(full_file_path, url):
                 # Error occurred while downloading file, mark job as failed and record error message
                 mark_job_status(job_id, "failed")
-                job = job_manager.getJobById(job_id)
-                file_type = job_manager.getFileType(job_id)
+                job = sess.query(Job).filter_by(job_id = job_id).one()
+                file_type = sess.query(Job).options(joinedload("file_type")).filter_by(job_id = job_id).one().file_type.name
```

Comment:
Similar here, we can just do `job.file_type.name` as `Job`s prefetch `file_type`.